scripts/wallet.js

```diff
@@ -250,7 +251,27 @@ export class Wallet {
         return !!this.#masterKey;
     }
 
-    async encryptWallet(strPassword) {
+    /**
+     * Check if the current encrypted keyToBackup can be decrypted with the given password
+     * @param {string} strPassword
+     * @return {Promise<boolean>}
+     */
+    async decrypt(strPassword) {
```

Comment:
The name is very misleading, this doesn't `decrypt` the wallet, it just checks if the password is correct. Considering that `encrypt` does encrypt the wallet, it's a confusing interface

Reply:
changed in `checkDecryptPassword`